### ISSUE TITLE
Would you be interested in a systemd service file to run this as a service?

### ISSUE THREAD

**Issue by txtsd** [xvzc/SpoofDPI#228]
### Description

I've created and included a hardened [systemd service file](https://aur.archlinux.org/cgit/aur.git/tree/spoofdpi.service?h=spoofdpi) in the AUR packages. It leverages a [conf.d file](https://aur.archlinux.org/cgit/aur.git/tree/spoofdpi.conf.d?h=spoofdpi) to pull some values without having to override the service, which was designed by @IcyColdified.

**Comment by harryqt** [comment]
I use this on Pi4 (Debian 12): `/etc/systemd/system/spoofdpi.service`

```
[Unit]
Description=SpoofDPI service
After=network.target

[Service]
Type=simple
User=harry
ExecStart=/home/harry/.spoofdpi/bin/spoofdpi -addr 0.0.0.0 -dns-addr 192.168.5.1 -banner=false -system-proxy=false
Restart=on-failure

[Install]
WantedBy=multi-user.target
```

**Comment by xvzc** [comment]
Yeah sounds great.

**Comment by txtsd** [comment]
What directory should I add it in?

**Comment by xvzc** [comment]
Will it bb just a single file? then it's fine to add it to the root directory

**Comment by txtsd** [comment]
It's two files.

**Comment by xvzc** [comment]
would it be okay to put them in /_assets/systemd ?

**Comment by txtsd** [comment]
Yes that seems okay for now. We can move them later when we handle packaging.
